packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts

```diff
@@ -5,6 +5,7 @@ import { generate } from '../JSONSchema';
 describe('JSONSchema generator', () => {
   const ipRegExp = /\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b/;
   const emailRegExp = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  const uuidRegExp = /[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$/;
```

Comment:
```suggestion
  const uuidRegExp = /^[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$/;
```
Above will ensure that prefix has been removed.

Reply:
added :+1:

Reply:
Is this comment still relevant, since we've removed all our custom code? @StefanDywersant